components/purchase_modal/purchase_modal.tsx

```diff
@@ -290,6 +318,72 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
         });
     }
 
+    paymentFooterText = () => {
+        const normalPaymentText = (
+            <FormattedMessage
+                defaultMessage={'Payment begins: {beginDate}'}
+                id={'admin.billing.subscription.paymentBegins'}
+                values={{
+                    beginDate: getNextBillingDate(),
+                }}
+            />
+        );
+
+        let payment = normalPaymentText;
+        if ((!this.props.isFreeTrial && this.state.currentProduct?.name.toLowerCase().includes('starter')) &&
+                !this.state.selectedProduct?.name.toLowerCase().includes('starter')) {
+            const announcementTooltip = (
```

Comment:
Just a few UI inconsistencies with the designs in the ticket
<img width="483" alt="Screenshot 2021-07-14 at 11 57 36" src="https://user-images.githubusercontent.com/28563179/125596935-e51211a1-e64a-4307-9785-8a121ec10c44.png">

- The color of the text after "Prorated Payments" looks different
- The date i.e. "Aug 1, 2021" should begin on a new line